packages/patternfly-4/react-core/README.md

```diff
@@ -104,4 +104,4 @@ yarn test packages/patternfly-4/react-core
 ```
 
 [patternfly-4]: https://github.com/patternfly/patternfly-next
-[docs]: https://patternfly-react.netlify.com/
+[docs]: http://patternfly-react.surge.sh/patternfly-4/index.html
```

Comment:
might be good to keep the `https`. Not that the docs site needs to be https, but it seems to be good practice to promote secure sites.